src/Windows/Avalonia.Direct2D1/Media/DrawingContextImpl.cs

```diff
@@ -104,29 +110,32 @@ public void Dispose()
         /// <param name="bitmapInterpolationMode">The bitmap interpolation mode.</param>
         public void DrawImage(IRef<IBitmapImpl> source, double opacity, Rect sourceRect, Rect destRect, BitmapInterpolationMode bitmapInterpolationMode)
         {
-            using (var d2d = ((BitmapImpl)source.Item).GetDirect2DBitmap(_renderTarget))
+            using (var d2d = ((BitmapImpl)source.Item).GetDirect2DBitmap(_deviceContext))
             {
                 var interpolationMode = GetInterpolationMode(bitmapInterpolationMode);
 
-                _renderTarget.DrawBitmap(
+                _deviceContext.DrawBitmap(
                     d2d.Value,
                     destRect.ToSharpDX(),
                     (float)opacity,
                     interpolationMode,
-                    sourceRect.ToSharpDX());
+                    sourceRect.ToSharpDX(),
+                    null);
             }
         }
 
-        private static SharpDX.Direct2D1.BitmapInterpolationMode GetInterpolationMode(BitmapInterpolationMode interpolationMode)
+        private static InterpolationMode GetInterpolationMode(BitmapInterpolationMode interpolationMode)
         {
             switch (interpolationMode)
             {
                 case BitmapInterpolationMode.LowQuality:
-                    return SharpDX.Direct2D1.BitmapInterpolationMode.NearestNeighbor;
+                    return InterpolationMode.NearestNeighbor;
                 case BitmapInterpolationMode.MediumQuality:
+                    return InterpolationMode.Linear;
                 case BitmapInterpolationMode.HighQuality:
+                    return InterpolationMode.HighQualityCubic;
                 case BitmapInterpolationMode.Default:
-                    return SharpDX.Direct2D1.BitmapInterpolationMode.Linear;
+                    return InterpolationMode.Linear;
```

Comment:
It seems to write the default nearly by medium quality that is means the default is medium quality.

Reply:
Default is the value that the underlying platform uses if you dont pass a value. In case of Direct2D this is linear Interpolation. This is mainly to make sure nothing changes in terms of image quality after this feature got introduced.